packages/salesforcedx-vscode-lwc-mobile/README.md

```diff
@@ -0,0 +1,27 @@
+# Lightning Web Components Preview on Mobile for Visual Studio Code
+
+This extension provides the ability to preview Lightning Web Components on an Android emulator or iOS simulator for testing purposes.
+
+**DO NOT INSTALL THIS EXTENSION DIRECTLY. Install the complete [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) instead.**
+
+## Documentation
+
+For documentation, visit the [Salesforce Extensions for Visual Studio Code](https://forcedotcom.github.io/salesforcedx-vscode) documentation site.
+
+## Bugs and Feedback
+
+To report issues with Salesforce Extensions for VS Code, open a [bug on GitHub](https://github.com/forcedotcom/salesforcedx-vscode/issues/new?template=Bug_report.md). If you would like to suggest a feature, create a [feature request on GitHub](https://github.com/forcedotcom/salesforcedx-vscode/issues/new?template=Feature_request.md).
+
+## Resources
```

Comment:
I guess we'll need to check with Product and Doc to decide what to link in here.